src/app/services/question.service.spec.ts

```diff
@@ -160,6 +160,82 @@ describe('QuestionService', () => {
     });
   });
 
+  describe('.searchForQuestionBy()', () => {
+
+    it('should return an array of matching question dtos if one with a matching title can be found', () => {
+      // Given
+      const questionNameToSearchFor = 'test-question';
+      const expectedSearchResults = {questions: [new QuestionDto({ title: questionNameToSearchFor, parentTopicTitle: 'nonsense' })]};
```

Comment:
This is the wrong shape of the data to be returned. this is fixed in PR https://github.com/math-dojo/single-page-web-app/pull/72 and caused some of the intermittent failures in the unit tests